addons/mail/models/mail_activity_plan.py

```diff
@@ -61,3 +64,85 @@ def copy_data(self, default=None):
             for plan, vals in zip(self, vals_list):
                 vals['name'] = _("%s (copy)", plan.name)
         return vals_list
+
+    def _get_summary_lines(self, plan_date=None):
```

Comment:
You can globally revert all that, we have a task to change that for the freeze, it is going to conflict and we will anyway revert everything I think.